app/controllers/submissions_controller.rb

```diff
@@ -1,8 +1,5 @@
 class SubmissionsController < ApplicationController
 
-  autolabRequire File.join(Rails.root, 'app/controllers/assessment/autograde.rb')
```

Comment:
I moved all of the autograding code out of submissions_controller.rb, so there's no need for it to be imported. (There aren't any functions in autograde that this calls, as far as I know).